core/shared/platform/windows/win_time.c

```diff
@@ -4,6 +4,56 @@
  */
 
 #include "platform_api_vmcore.h"
+#include <winternl.h>
+
+
+
+#define NANOSECONDS_PER_SECOND 1000000000
+
+typedef uint32_t __wasi_clockid_t;
+#define __WASI_CLOCK_REALTIME (0)
+#define __WASI_CLOCK_MONOTONIC (1)
+#define __WASI_CLOCK_PROCESS_CPUTIME_ID (2)
+#define __WASI_CLOCK_THREAD_CPUTIME_ID (3)
+
+typedef uint64_t __wasi_timestamp_t;
```

Comment:
I think we don't need this typedef, you can use `uint64` when needed.

Reply:
no, I think we can remove the typedef from this file, and replace `__wasi_timestamp_t` usage with the `uint64`